.github/actions/run-android-device-farm-sync-test/action.yml

```diff
@@ -0,0 +1,72 @@
+name: 'Run Android tests on Device Farm'
```

Comment:
Unless I am missing it, the only difference between this action and `run-android-device-farm-test` seems to be the package name: `io.realm.sync.testapp.test` vs. `io.realm.testapp.test`. Isn't there a way we can combine them?

I guess `app_auxiliary_files` is also a difference, but couldn't we just make that optional?

Reply:
Updated with common action used for sync and non-sync device farm tests